dhis-2/dhis-support/dhis-support-external/src/main/java/org/hisp/dhis/external/conf/ConfigurationKey.java

```diff
@@ -75,6 +75,12 @@ public enum ConfigurationKey {
   /** Node identifier, optional, useful in clusters. */
   NODE_ID("node.id", "", false),
 
+  /**
+   * When true, the node will unconditionally set its node ID during leader election causing it to
+   * win the election as long as it is alive.
```

Comment:
I know this isn't (yet) end user documentation but I would suggest using the word "cluster" somewhere just so that people not concerned with clusters will skip over it faster.